test/test_main.cpp

```diff
@@ -1,19 +1,54 @@
 #define CATCH_CONFIG_MAIN // This tells Catch to provide a main() - only do this in one cpp file
 #include "../src/dht_11.hpp"
```

Comment:
Adding "../src/" shouldn't be necessary as every file in the /src/ directory can be included by just using the file name.